common/src/main/java/com/viaversion/viaversion/protocols/v1_12_2to1_13/data/MappingData1_13.java

```diff
@@ -81,6 +81,14 @@ protected void loadExtras(final CompoundTag data) {
             blockMappings.setNewId(1557, 3986); // chiseled stone bricks
         }
 
+        // Don't apply VB only mappings for purple/not-colored shulker boxes
+        // TODO Check which blocks might also be affected by inverting the VB mappings for VV
```

Comment:
This isn't related to VB, it's how they were done for VV initially and mapped automatically because it *is* called purple shulker box in 1.12